packages/hoppscotch-common/src/helpers/keybindings.ts

```diff
@@ -132,17 +133,16 @@ function getPressedKey(ev: KeyboardEvent): Key | null {
   if (isLetter) return ev.code.toLowerCase().substring(3) as Key
 
   // Check if number keys
-  if (val.length === 1 && !isNaN(val as any)) return val as Key
-
-  // Check if question mark
-  if (val === "?") return "?"
+  const isDigit = ev.code.toLowerCase().startsWith("digit")
+  if (isDigit) return ev.code.toLowerCase().substring(5) as Key
```

Comment:
```suggestion
  const isDigit = val.startsWith("digit")
  if (isDigit) return val.substring(5) as Key
```

Reply:
@jamesgeorge007  I have made the changes.